doc/sphinx_ext.rst

```diff
@@ -0,0 +1,19 @@
+Sphinx Extension
+================
+Some of Gossip depended packages uses hooks as part of their public API. As such, adding the hooks to the documentation is needed.
```

Comment:
Some packages depending on gossip use hooks as part of their public APIs.

Reply:
In such cases, adding hook information to their respective documentation pages may be needed.